pkg/resources/environment.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// sanitizeEnvVar will take the value of a environment variable and sanatises it.
```

Comment:
Nit, fix in a follow-up:

```suggestion
// SanitizeEnvVar will take the value of an environment variable and sanitize it.
```